src/lib/components/modal/ModalContent.svelte

```diff
@@ -0,0 +1 @@
+<p class="text-lg"><slot /></p>
```

Comment:
Does it make sense to get rid of these one-liner components in favor of named slots?

Something like this - https://svelte.dev/repl/207ce84ff7c54091a72358cfde64e964?version=3.48.0

Reply:
That is something I considered, but I went with the component approach for a few reasons:
- We already use the component approach for tooltips (e.g. Tooltip, TooltipContent, TooltipTitle)
- I used Vercel's component library Modal API as inspiration, see: https://vercel.com/design/modal
- I think it would be easier for new teammates / other developers to onboard by browsing our component library and seeing the family of Modal components, rather than having to inspect the Modal code and search for named slots